src/server.cpp

```diff
@@ -58,6 +58,8 @@ CHighPrecisionTimer::CHighPrecisionTimer ( const bool bNewUseDoubleSystemFrameSi
     veciTimeOutIntervals[1] = 1;
     veciTimeOutIntervals[2] = 0;
 
+    setObjectName ( "CHighPrecisionTimer" );
```

Comment:
I don't think we need this for Windows.

Reply:
I wonder what QThread does on Windows... and how you monitor threads -- surely there's some way?

Reply:
We should focus on Linux for now. Interestingly, even without your patch I get the information about the highprecisiontimer in my htop:
![grafik](https://user-images.githubusercontent.com/46655886/89574027-ccd62100-d82b-11ea-9789-a35dd981e049.png)
So maybe there is no need to set it at all (even not for Linux).

Reply:
Ah... maybe it's something like this:
```
QObject qObject; // or whatever
QThread qThread;
qObject->moveToThread(qThread);
qThread->start();
```
will give you a default thread name of `QThread` (your thread's class).  But `CHighPrecisionTimer` extends `QThread`, more like (from memory):
```
CHighPrecisionTimer timerThread;
timerThread.start();
```
So I guess my setObjectName here to the same thing is rather redundant :).